<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secondary Structure Calc Python</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Logo_Icons/SS-Icon.png</normaloff>Logo_Icons/SS-Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonStartCalc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>355</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Calc</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Logo_Icons/SS-Icon.png</normaloff>Logo_Icons/SS-Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>34</width>
      <height>34</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Message label</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Logo_Icons/Spectrum100.png</normaloff>Logo_Icons/Spectrum100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>34</width>
      <height>34</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMyProteinFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>labelMyProteinFile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMyProteinDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>195</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>labelMyProteinDir</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>246</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>For CD calculations, please select Ref set and origin</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>100</width>
      <height>39</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonRefSetSP175">
     <property name="text">
      <string>Ref Set SP175</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRefSetSMP180">
     <property name="text">
      <string>Ref Set SMP180</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>155</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Protein file and folder</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShouldIPlot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot calculated spectra</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShouldIAlsoPlotSelcon2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Also plot Selcon2 calculated spectra</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>174</width>
      <height>39</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonRefSetAU">
     <property name="text">
      <string>Ref set curated by AU (PCDDB)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRefSetBBK">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Ref set BBK (Prof Wallace)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Which method:</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonSelcon3">
     <property name="text">
      <string>Selcon3Py</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCDSSTR">
     <property name="text">
      <string>CDSSTRPy</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonASCIIXY">
     <property name="text">
      <string>File is XY ASCII (.txt .dat)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCSVXY">
     <property name="text">
      <string>File is XY CSV (.csv)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxScaleCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale CD (0.9-1.1 allowed)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditScaleFactor">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IR Abs max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Logo_Icons/SScalcPyLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chose spectroscopic method</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonCD">
     <property name="text">
      <string>CD file and calculations</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonIR">
     <property name="text">
      <string>IR file and calculations</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCDIR">
     <property name="text">
      <string>Combined CD  and IR file and calculations</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Combined CD and IR:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIRAbsMax">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxUseCDIR2cm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use 2cm-1 data in CD-IR data set</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>pushButtonStartCalc</zorder>
   <zorder>labelMessage</zorder>
   <zorder>pushButtonSelectFile</zorder>
   <zorder>labelFileName</zorder>
   <zorder>labelMyProteinFile</zorder>
   <zorder>labelMyProteinDir</zorder>
   <zorder>label</zorder>
   <zorder>splitter_2</zorder>
   <zorder>splitter_3</zorder>
   <zorder>label_2</zorder>
   <zorder>checkBoxShouldIPlot</zorder>
   <zorder>splitter</zorder>
   <zorder>label_3</zorder>
   <zorder>splitter_4</zorder>
   <zorder>splitter_5</zorder>
   <zorder>checkBoxScaleCD</zorder>
   <zorder>lineEditScaleFactor</zorder>
   <zorder>label_4</zorder>
   <zorder>label_7</zorder>
   <zorder>label_6</zorder>
   <zorder>checkBoxShouldIAlsoPlotSelcon2</zorder>
   <zorder>label_8</zorder>
   <zorder>lineEditIRAbsMax</zorder>
   <zorder>line</zorder>
   <zorder>checkBoxUseCDIR2cm</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
